resources/views/layout.blade.php

```diff
@@ -25,6 +25,7 @@
     {!! HTML::style('//fonts.googleapis.com/css?family=Yellowtail') !!}
     {!! HTML::style('//fonts.googleapis.com/css?family=Josefin+Slab:600') !!}
     {!! HTML::style(elixir("css/app-all.css")) !!}
+    @yield('styles')
```

Comment:
We shouldn't have any additional styles here - everything should go in app-all.css, which is built by gulp